### ISSUE TITLE
Error on android build

### ISSUE THREAD

**Issue by hypery2k** [bradmartin/nativescript-drawingpad#35]
With 2.1.1 I'm getting the following error:
```
* What went wrong:
Could not compile script '.../node_modules/nativescript-drawingpad/platforms/android/include.gradle'.
> startup failed:
  script '.../node_modules/nativescript-drawingpad/platforms/android/include.gradle': 1: Invalid variable name. Must start with a letter but was:
  . At [1:1]  @ line 1, column 1.
     //default elements
     ^
```

**Comment by bradmartin** [comment]
published 3.0.2 with fix

**Comment by hypery2k** [comment]
thx
